pkg/proctitle/proctile.go

```diff
@@ -0,0 +1,14 @@
+//go:build linux
+// +build linux
+
+package proctitle
+
+import (
+	"os"
+
+	"github.com/erikdubbelboer/gspt"
+)
+
+func SetProcTitle(cmd string) {
+	gspt.SetProcTitle(os.Args[0] + " agent")
+}
```

Comment:
Is this a transitional API? We don't do anything with the `cmd`?

Reply:
oh I think that was a mistake, fixing now